packages/discord.js/src/util/DataResolver.js

```diff
@@ -67,7 +67,7 @@ class DataResolver extends null {
       return image;
     }
     const file = await this.resolveFile(image);
-    return DataResolver.resolveBase64(file);
+    return this.resolveBase64(file);
```

Comment:
The method is static
```suggestion
    return DataResolver.resolveBase64(file);
```

Reply:
hm, just standardizing across the file. every other other static->static reference is using `this`. should the rest of the file be swapped over the other way then? as DataResolver is just a container for static methods, there isn't really a binding problem here.

Reply:
> The method is static
> ```suggestion
>     return DataResolver.resolveBase64(file);
> ```

`this` is `DataResolver` here so no need of this change...

Reply:
> just standardizing across the file. every other static->static reference is using `this`

Didn't notice the rest of the static methods were using `this`, you can ignore this suggestion 

> should the rest of the file be swapped over the other way then

IMO, yes, but that's not up to me